evm/vm/code_stream.py

```diff
@@ -1,3 +1,7 @@
+from typing import (  # noqa: F401
+    Iterator,
+    Set
+)
```

Comment:
There should be no blank lines between stdlib imports, and they should be ordered alphabetically